### ISSUE TITLE
How did you clone enyojs/ares-webos-sdk?

### ISSUE THREAD

**Issue by sparkleholic** [stevenvong/ares-webos-sdk#3]
I'm a maintainer of ares-webos-sdk which is a private repository.
How did you clone the private repository?

**Comment by sparkleholic** [comment]
Sorry, I've just checked that you had submitted minified sources here.

**Comment by stevenvong** [comment]
YOu're right. I was working on a project related to this and felt that wanted an easier way to access the webos sdk.
